core/templates/dev/head/directives.js

```diff
@@ -164,3 +164,50 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('handleDropdownNavigation', function () {
+  return {
+    restrict: 'A',
+    link: function (scope, elm, attrs) {
+      elm.on('focus', function (e) {
+        elm.bind('keydown keypress', function (e) {
+          if (e.keyCode === 13) {
+            // open the submenu on pressing enter
+            e.preventDefault();
+            if (!elm.closest('li').hasClass('open')) {
```

Comment:
Using ng-class would be better than this; can we try that?

Reply:
One possible approach is to have a directive for the menu that stores the open/close toggle state. Then your li element can do something like ng-class="{'open': xxxMenuIsOpen}", where xxxMenuIsOpen is the value stored in the directive controller. Maybe?

Reply:
oh right!Implementing this, we can eliminate addClass and removeClass methods in the directive controller and just toggle a variable like activeMenu = 'aboutMenu.'

Reply:
Yep -- if you can use (or reuse) the same directive in multiple places, so much the better. Have a go at that, and let's see how it goes?

Reply:
Done, eliminated usage of addClass and removeClass methods, and used ng-class.PTAL.